resources/assets/lib/components/report-form.coffee

```diff
@@ -95,15 +100,17 @@ export class ReportForm extends PureComponent
         className: "#{bn}__row"
         textarea
           className: "#{bn}__textarea"
+          onChange: @handleCommentsChange
           placeholder: osu.trans 'users.report.placeholder'
           ref: @textarea
```

Comment:
`sendReport` should be updated to use state and this ref variable can be removed